package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "scripts": {
     "fmt:global": "prettier --write .",
-    "fmt:global-check": "prettier --check ."
+    "fmt:global-check": "prettier --check .",
+    "postinstall": "npm rebuild --skip-scripts @safe-global/safe-4337-local-bundler"
```

Comment:
This looks weird, bit it is a work around to re-link the `bin` from the workspace package. The issue is that the JS file that is referenced in the `bin` field from the `package.json` is **generated** by the build step, and NPM can't figure out how to link it correctly.

The work-around is to manually re-build the package, this forces NPM to regenerate `bin` links. Note that, by the time the `postinstall` step happens, the package is guarateed to be built, as it is a dependency of other packages in the workspace.